src/gui/cli/Player.cpp

```diff
@@ -20,6 +21,19 @@ namespace Zappy::GUI {
         _orientation = aOrientation;
     }
 
+    void Player::setOrientation(unsigned long int aOrientation)
+    {
+        if (aOrientation == 1) {
```

Comment:
Orientation is an enum so you don't need to use if statements.
Because the first item of the enum (NORTH) corresponds to 0, we can deduce that `_orientation = aOrientation - 1`.

Reply:
I don't think `aOrientation` needs to be a `long`: its value goes from 1 to 4.

Reply:
You should also use a condition to make sure `aOrientation` stands between 1 and 4.